module/spring-boot-flyway/src/test/java/org/springframework/boot/flyway/autoconfigure/FlywayAutoConfigurationTests.java

```diff
@@ -897,6 +897,13 @@ void ignoreMigrationPatternsCorrectlyMapped() {
 				.containsExactly(ValidatePattern.fromPattern("*:missing")));
 	}
 
+	@Test
+	void ignoreMigrationPatternsIsFutureByDefault() {
+		this.contextRunner.withUserConfiguration(EmbeddedDataSourceConfiguration.class)
+			.run((context) -> assertThat(context.getBean(Flyway.class).getConfiguration().getIgnoreMigrationPatterns())
+				.containsExactly(ValidatePattern.fromPattern("*:future")));
+	}
+
```

Comment:
This tests Boot's default, but doesn't test that the default is aligned with Flyway's default.

We should keep this test, but also add two more:

- One in this class as originally proposed. It should test that setting the property to an empty string results in an empty list
- An extra assertion in `FlywayPropertiesTests.defaultValuesAreConsistent()` for ignore migration patterns.